ebml_matroska.xml

```diff
@@ -308,28 +308,28 @@ this can be used for tracks containing only translations of foreign-language aud
 See (#default-track-selection) for more details. (1 bit)</documentation>
     <extension cppname="TrackFlagForced"/>
   </element>
-  <element name="FlagHearingImpaired" path="\Segment\Tracks\TrackEntry\FlagHearingImpaired" id="0x55AB" type="uinteger" range="0-1" default="0" maxOccurs="1">
-    <documentation lang="en" purpose="definition">Set if that track is suitable for users with hearing impairments. (1 bit)</documentation>
+  <element name="FlagHearingImpaired" path="\Segment\Tracks\TrackEntry\FlagHearingImpaired" id="0x55AB" type="uinteger" range="0-1" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set to 1 resp. 0 if that track is suitable resp. unsuitable for users with hearing impairments.</documentation>
```

Comment:
Not a fan of `resp.`, I had to look up online to find out what you mean. I think it's `respectively`. We don't need to compress words here.

Reply:
Even with the words expanded the text is still hard to process. I would prefer something like

> Set to 1 if that track is suitable for users with hearing impairments, 0 otherwise.

Or even

> Set to 1 if that track is suitable for users with hearing impairments, 0 if unsuitable for users with hearing impairments.

Which clearly says if the value is explicitly set to 0 it's not suitable.

Reply:
> Set to 1 if that track is suitable for users with hearing impairments, 0 if unsuitable for users with hearing impairments.

I like this wording very much as it's much clearer that 0 is different from the element not being present. @mkver please adjust all descriptions this way. Thanks.

Reply:
I force-pushed a more explicit version. (Apologies for not knowing that "resp." is not a well-known abbreviation. It is in German.)